src/EventDriven.SchemaRegistry.Abstractions/EventDriven.SchemaRegistry.Abstractions.csproj

```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net5.0</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-        <Version>1.0.0-beta2</Version>
+        <Version>1.0.0-beta3</Version>
```

Comment:
Let me take care of the package version bump. I may move it out of beta at this time anyway.